src/libs/colors.js

```diff
@@ -1,6 +1,6 @@
 import Color from 'color'
 import _ from 'lodash/fp'
-import { isAnvil, isBaseline, isBioDataCatalyst, isDatastage, isElwazi, isFirecloud, isProjectSingular, isRareX, isTerra } from 'src/libs/config'
+import { isAnvil, isBaseline, isBioDataCatalyst, isDatastage, isElwazi, isFirecloud, isProjectSingular, isRareX, isTerra } from 'src/libs/brand-utils'
```

Comment:
What do you think about moving the color palette into the brands data structure?

I'm thinking that in general whenever "isX" is called, it means something is missing from the brands data structure.

Reply:
I agree with you and was envisioning it as part of the follow-up work.